### PR TITLE
Add support for multiple form switch indicator labels & adjust styling

### PR DESCRIPTION
**Pull request by kaankabalak**
Previously, only a single label was being displayed regardless of the
checked status of a switch.

This commit also adjusts form switch styling based on mockups, such as
greying out the switch track color when switch is not checked

Screen Shot 2020-11-10 at 1 34 22 PM

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx [hunk 1]
```diff
@@ -142,14 +142,21 @@ const StyledSwitch = withStyles({
...
     },
+    "&:hover": {
+      backgroundColor: "#fff",
+    }
   },
   checked: {},
   track: {
     height: 15,
-    backgroundColor: "#081C42",
+    backgroundColor: "#9C9C9C",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx:152]: I think colors must be inverse as in design:

Screen Shot 2020-11-10 at 8 22 37 PM
Screen Shot 2020-11-10 at 8 22 30 PM

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx (hunk 1 above)
Comment (kaankabalak) [reply portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx:152]: Will do @bexsoft 👍

**Review by Alevsk — APPROVED** [review]

File: portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx:152]: image

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx:152]: Screen Shot 2020-11-19 at 11 55 53 AM
I think how we have them right now makes sense, only the highlight color is off

**Comment by dvaldivia** [comment]
The changes are already merged

File: portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx (hunk 1 above)
Comment (kaankabalak) [reply portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx]: @dvaldivia, I have just rebased the branch, however, could you elaborate on what the highlight color should be please?

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper.tsx]: I think you have the right color in general, what we have in the design looks slightly counter intuitive to what we see in the design, after rebasing the branch, this PR shows there's 0 changes, so I assume the changes are already merged via other branches

**Pull request closed without merging**